dev-libs/libusb/patches/libusb-1.0.23.patchset

```diff
@@ -0,0 +1,59 @@
+From 5aa523289f82d1c7f01dde601356214920646542 Wed Jul 31 00:00:00 2019
+From: Ken Mays <kmays2000@gmail.com>
+Date: Thu, Sep 5 2019 01:15:08 +0000
+Subject: Fix Haiku port
```

Comment:
Were all the other patches upstreamed?

Reply:
I upstreamed what we had in the previous version, yes.